style.css

```diff
@@ -31,7 +31,7 @@ html{
     font-family: 'Roboto', sans-serif;
     font-size: 100%;
     color: var(--black-color);
-    background-color: var(--lynx-white-color);
+    background-color: var(--white-color);
 }
 
 button{
```

Comment:
empty declaration can be removed

Reply:
can you please explain ? I didn't understand.